packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/RepeatableComponent/index.js

```diff
@@ -14,6 +14,11 @@ const TypographyMaxWidth = styled(Typography)`
   max-width: 500px;
 `;
 
+const SimpleMenuAdapted = styled(SimpleMenu)`
+  margin-left: -6px;
```

Comment:
Why do we need this? Is there any other way rather than using negative margins with hardcoded values?

cc/ @HichamELBSI

Reply:
@gu-stav Can't we reset the `padding` instead of negative margin as we'd said?

Reply:
@soupette I've discussed this with Hicham option 2.

Here are the cases:

1. `SimpleMenu` without reducing padding-left

It looks really off.

<img width="1157" alt="Screenshot 2022-02-23 at 17 02 49" src="https://user-images.githubusercontent.com/2244375/155358192-cfc70684-b66f-4bea-a457-963b501ae44e.png">

2. `SimpleMenu` with reduced padding

<img width="1157" alt="Screenshot 2022-02-23 at 17 03 10" src="https://user-images.githubusercontent.com/2244375/155358274-9c178506-a5ae-486a-bf69-f15a66375708.png">

3. `SimpleMenu` with reduced padding and negative margin

<img width="1157" alt="Screenshot 2022-02-23 at 17 03 57" src="https://user-images.githubusercontent.com/2244375/155358391-00957770-dcac-45d1-b438-722ea365c300.png">

That's being said: I'm not a fan of negative margins either, but I think we have to properly align the menu somehow. Reducing the padding already works, but it looks still a bit off.

Also pinging @maevalienard here.

Reply:
Another idea: we could create, similar to button, a `size="S|M"` variant. In "s" we then could reduce the padding to a minimum ... What do you think about it?

Reply:
Great idea! Since we are using this component in several context, it should be customizable

Reply:
I've opened a PR here: https://github.com/strapi/design-system/pull/552 and leave this discussion open until merged.

The question if we agree on using a negative margin here still needs to be discussed.

Reply:
Well if you really need the negative margin I'm fine with it however can you use the sizes from the DS instead of hardcoding `6px` so it still consistent?

Reply:
@soupette I'll do my best :)